INSTALL.rst

```diff
@@ -18,6 +18,11 @@ Windows and Linux distributions::
   python -m pip install -U pip
   python -m pip install -U matplotlib
 
+If you are unexpetedly compiling Matplotlib from source (rather than
```

Comment:
What does the "unexpectedly" mean here? Don't we expect somebody is compiling Matplotlib?

I suppose you want to express something like

> If this command results in Matplotlib being compiled from source and there's trouble with the compilation, you can add `--prefer-binary` to select the newest version of Matplotlib for which there is a precompiled wheel.